code/renderers/solid/template/stories/html-mdx.stories.mdx

```diff
@@ -0,0 +1,21 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs';
```

Comment:
As stories defined in MDX is being deprecated in 7.0 I suggest you remove this template file.